src/_scss/components/typeahead/_typeahead.scss

```diff
@@ -24,11 +24,33 @@
             right: 0px !important;
         }
     }
-    
+
     @import "./_dropdownList";
 
     input {
         @import "../inputs/_generic";
         @include genericInput;
     }
-}
\ No newline at end of file
+    
+    // REPLACE RESPONSIVE WITH BOOTSTRAP IN THE FUTURE
```

Comment:
<img width="906" alt="screen shot 2018-10-29 at 2 47 56 pm" src="https://user-images.githubusercontent.com/7108785/47672988-ba748100-db89-11e8-9e2f-606d5141a0ee.png">
Because this is shared code, it is causing the typeahead to shrink unnecessarily elsewhere on the site - could you move this code to `pages/dashboard/_typeaheadFilter.scss`?